error "don't have function for random number generator"

```
g++ -DHAVE_CONFIG_H -I. -I. -O3 -pipe -DNDEBUG -pedantic-errors -Wparentheses -Wreturn-type -Wcast-qual -Wall -Wpointer-arith -Wwrite-strings -Wconversion -Wno-unknown-pragmas -Wno-long-long -DIPOPT_BUILD -MT IpUtils.lo -MD -MP -MF .deps/IpUtils.Tpo -c IpUtils.cpp  -fPIC -DPIC -o .libs/IpUtils.o
IpUtils.cpp:178:5: error: #error "don't have function for random number generator"
 #   error "don't have function for random number generator"
     ^
IpUtils.cpp:195:5: error: #error "don't have function for random number generator"
 #   error "don't have function for random number generator"
     ^
IpUtils.cpp: In function 'Ipopt::Number Ipopt::IpRandom01()':
IpUtils.cpp:182:3: warning: no return statement in function returning non-void [-Wreturn-type]
   }
   ^
make[2]: *** [IpUtils.lo] Error 1
make[2]: Leaving directory `/home/rick/.julia/v0.4/Ipopt/deps/src/Ipopt-3.11.7/Ipopt/src/Common'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/home/rick/.julia/v0.4/Ipopt/deps/src/Ipopt-3.11.7/Ipopt'
make: *** [install-recursive] Error 1
================================[ ERROR: Ipopt ]================================

failed process: Process(`make install`, ProcessExited(2)) [2]
while loading /home/rick/.julia/v0.4/Ipopt/deps/build.jl, in expression starting on line 81

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: Ipopt had build errors.

 - packages with build errors remain installed in /home/rick/.julia/v0.4
 - build a package and all its dependencies with `Pkg.build(pkg)`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated
```

--- a/deps/build.jl
+++ b/deps/build.jl
@@ -43,7 +43,7 @@ provides(SimpleBuild,
                 ChangeDirectory(joinpath(srcdir,"ThirdParty","Mumps"))
                 `./get.Mumps`
             end
-            `./configure --prefix=$prefix --enable-dependency-linking --with-blas=$prefix/lib/libcoinblas.a --with-lapack=$prefix/lib/libcoinlapack.a`
+            `./configure --prefix=$prefix --enable-dependency-linking coin_skip_warn_cxxflags=yes --with-blas=$prefix/lib/libcoinblas.a --with-lapack=$prefix/lib/libcoinlapack.a`
             `make install`
         end
     end),libipopt, os = :Unix)

